src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -429,7 +584,7 @@ class AlterTableActionSegment(BaseSegment):
             Ref("ColumnReferenceSegment"),
             Ref("DatatypeSegment"),
             Sequence("COLLATE", Ref("QuotedLiteralSegment"), optional=True),
-            Ref("ColumnOptionSegment", optional=True),
+            AnyNumberOf(Ref("ColumnConstraintSegment", optional=True)),
```

Comment:
Do we need `optional=True` for `AnyNumberOf`?

Reply:
Nope, removed!